galeracluster/source/dealingwithmultimasterconflicts.rst

```diff
@@ -24,36 +24,82 @@ Galera Cluster can cope with a situation such as this by using Certification Bas
-	wsrep_local_cert_failures
+     +-----------------------+-------+
+     | Variable_name         | Value |
+     +-----------------------+-------+
+     | wserp_local_bf_aborts | 333   |
+     +-----------------------+-------+
+          
+     SHOW STATUS LIKE 'wsrep_local_cert_failures';
 
-Setting the ``wsrep_provider_options="cert.log_conflicts=1"`` will log messages like::
+     +---------------------------+-------+
+     | Variable_name             | Value |
+     +---------------------------+-------+
+     | wserp_local_cert_failures | 333   |
+     +---------------------------+-------+
+     
+  :ref:`wsrep_local_bf_aborts <wsrep_local_bf_aborts>` gives the total number of local transactions aborted by slave transactions while in execution.  :ref:`wsrep_local_cert_failures <wsrep_local_cert_failures>` gives the total number of transactions that have failed certification tests.
 
-     7:51:13 [Note] WSREP: trx conflict for key (1,FLAT8)056eac38 0989cb96: source: cdeae866-d4a8-11e3-bd84-479ea1a1e941 version: 3 local: 1 state: MUST_ABORT flags: 1 conn_id: 160285 trx_id: 29755710 seqnos (l: 643424, g: 8749173, s: 8749171, d: 8749171, ts: 12637975935482109) <--X--> source: 5af493da-d4ab-11e3-bfe0-16ba14bdca37 version: 3 local: 0 state: APPLYING flags: 1 conn_id: 157852 trx_id: 26224969 seqnos (l: 643423, g: 8749172, s: 8749171, d: 8749170, ts: 12637839897662340)
+- Lastly, you can enable the wsrep Provider option :ref:`cert.log_conflicts <cert.log_conflicts>` in the configuration file.
+
+  .. code-block:: ini
+
+     wsrep_provider_options="cert.log_conflicts=YES"
+
+  This tells the node to log information specifically on certification failures.  For example, it might log a message like this:
+
+  .. code-block:: text
+		  
+     7:51:13 [Note] WSREP: trx conflict for key (1,FLAT8)056eac38 0989cb96:
+     source: cdeae866-d4a8-11e3-bd84-479ea1a1e941 version: 3 local: 1 state:
+     MUST_ABORT flags: 1 conn_id: 160285 trx_id: 29755710 seqnos (l: 643424,
+     g: 8749173, s: 8749171, d: 8749171, ts: 12637975935482109) <--X-->
+     source: 5af493da-d4ab-11e3-bfe0-16ba14bdca37 version: 3 local: 0 state:
+     APPLYING flags: 1 conn_id: 157852 trx_id: 26224969 seqnos (l: 643423,
+     g: 8749172, s: 8749171, d: 8749170, ts: 12637839897662340)
 
 ------------------------------
- Autocommitting Transactions
+ Auto-committing Transactions
 ------------------------------
 
 .. index::
    pair: Parameters; wsrep_retry_autocommit
 
-When a conflicting transaction is rolled back, the client application sees a deadlock error. The client application *should* try to re-commit the deadlocked transactions, but not all client applications have this logic inbuilt.
+Transactions that come into conflict are rolled back by the cluster.  The client application registers this rollback as a deadlock error.  Ideally, the client application *should* re-commit the deadlocked transaction, but not all client applications have this logic built in.

```

Comment:
Only one, the later, of two conflicting transactions is rolled back, not both, hence the use of singular "transaction" in the original text.

Also, "re-commit" is not correct as it implies that only the COMMIT operation needs to be reissued. Instead, the entire transaction needs to be retried, so "retry" would be more appropriate.